roles-and-groups.md

```diff
@@ -0,0 +1,153 @@
+In most cases a project is a single GitHub repository, but it can also be a group of repositories that are related to each other.
+A person can have multiple roles in multiple projects.
+
+### Contributors
+
+Any person who contributes to a project is a Contributor. A contribution can be anything from opening an issue, to writing code, to writing documentation, etc.
+
+  - If a pull request has not received an approval within 2 business days or a discussion has no activity for more than 2 business days, the owner must reach out to the maintainers via our [hiero-general Discord channel](https://discord.lfdecentralizedtrust.org/).
+  - If a pull request or discussion has gone stale for more than 2 weeks, the repo maintainers may close them due to inactivity.
+
+We honor legitimate contributions to the project, and we want to make sure that all contributions are recognized.
+
+A Contributor can be nominated by a Committer or a Maintainer to become a Committer:
+
+  - **Start Contributing**: Submit pull requests, report bugs, improve documentation, review code. Show consistency and quality in your contributions.
+  - **Build Trust and Track Record**: Active committers will notice your contributions, your understanding, and responsiveness.
+  - **Get noticed**: Consistent and valuable contributions will demonstrate more project involvement and reliability.
+  - **Review eligibility**: A minimum number of approved PRs and a reasonable time contributing to the project (often 3-6 months) is the general acceptable baseline for eligibility.
+
+### Committers
+
+A Committer has specific rights for the project.
+
+As an example, a Committer can assign people to issues, merge PRs, or create branches directly in a repository of the project.
+Any Committer or Maintainer of the given project can nominate a Contributor to become a Committer based on the project's needs and following these steps:
+
+- **Define your project's needs**: A repository should have minimum 3 committers to ensure a project's sustainability and resilience. It will also allow for fairness in the votes. Project owners can choose to have more committers depending on the size of their project.
+- **Evaluate your project's performance**: Committers and Maintainers should monitor their project health and velocity and make decisions about whether the project needs more Committers.
+- **Verify your project's needs**: Committers and Maintainers should define the number of good contributions that a Contributor must reach to consider a nomination. This is based on the size and complexity of the project in question.
+- **Nominate new committers**: To promote a contributor, a vote should be held in a GitHub PR (by proposing a change to the config.yaml file) and by describing the basis reached for nominating the Contributor. The template [vote_pr_template.md](https://github.com/hiero-ledger/governance/blob/main/.github/PULL_REQUEST_TEMPLATE/vote_pr_template.md) should be used to creating a nomination.
```

Comment:
We should have 1 source for the full description of a voting process and not 2. Best would be to add the "Voting" section from the PR template in this doc

Reply:
The reason why there are 2 is because we have one addressed to the Committers and one addressed to the Maintainers. Having them separate can offer more flexibility in the long run since the work of the committers and maintainers is different. 

In general, the doc is categorized by roles.